app/helpers/hyku_helper.rb

```diff
@@ -19,4 +19,8 @@ def admin_host?
   def admin_only_tenant_creation?
     ActiveModel::Type::Boolean.new.cast(ENV.fetch('HYKU_ADMIN_ONLY_TENANT_CREATION', false))
   end
+
+  def turbolinks_enabled?
+    ActiveModel::Type::Boolean.new.cast(ENV.fetch('HYKU_ALLOW_REDIRECT_TO_VALKYRIE_RESOURCE', true))
```

Comment:
When our ENV var is `true`, turbolinks needs to be `false`, and vice versa. Here's my suggestion on how we can do that in a (hopefully) easy-to-understand, descriptive way: 

```suggestion
  def block_valkyrie_redirect?
    !ActiveModel::Type::Boolean.new.cast(ENV.fetch('HYKU_ALLOW_REDIRECT_TO_VALKYRIE_RESOURCE', true))
```

```mermaid
flowchart TD
A1(HYKU_ALLOW_REDIRECT_TO_VALKYRIE_RESOURCE)
A2[block_valkyrie_redirect?]
A3[turbolinks: false]
A4(Page redirects)
B1(HYKU_ALLOW_REDIRECT_TO_VALKYRIE_RESOURCE)
B2[block_valkyrie_redirect?]
B3[turbolinks: true]
B4(Page does not redirects)

A1 --> |true|A2 --> |false|A3 --> A4
B1 --> |false|B2 --> |true|B3 --> B4
```

Reply:
would it be semantically different if we used

```rb
    ActiveModel::Type::Boolean.new.cast(ENV.fetch('HYKU_ALLOW_REDIRECT_TO_VALKYRIE_RESOURCE', false))
```

Reply:
so `false` instead of not `true`

Reply:
I don't think so. The second argument of `ENV.fetch` is a fallback value:

```ruby
# If a is nil, use b
ENV.fetch(a, b)
```

So the value should be whatever we want the default to be, which in this case, I think is `true`

Reply:
alright sounds good to me